src/lib/utils/thorchain/lp.ts

```diff
@@ -233,34 +213,23 @@ export const estimateRemoveThorchainLiquidityPosition = async ({
 }
 
 export const calculateEarnings = (
-  _assetLiquidityFees: string,
-  _runeLiquidityFees: string,
+  pool: MidgardEarningsHistoryPoolItem,
   userPoolShare: string,
   runePrice: string,
-  assetPrice: string,
 ) => {
-  const assetLiquidityFees = fromThorBaseUnit(_assetLiquidityFees)
-  const runeLiquidityFees = fromThorBaseUnit(_runeLiquidityFees)
-
-  const userShare = bn(userPoolShare)
-  const assetEarnings = userShare.times(assetLiquidityFees).times(2).toFixed()
-  const runeEarnings = userShare.times(runeLiquidityFees).times(2).toFixed()
+  const totalEarningsRune = fromThorBaseUnit(pool.earnings).times(userPoolShare)
+  const totalEarningsFiat = totalEarningsRune.times(runePrice).toFixed()
```

Comment:
i.e not in USD
```suggestion
  const totalEarningsFiatUserCurrency = totalEarningsRune.times(runePrice).toFixed()
```

Reply:
Similar comment here. Isn't `Fiat` generic? Just feels like `UserCurrency` is redundant imo